VT-3 Preset Expansion/index.js

```diff
@@ -0,0 +1,450 @@
+var easymidi = require('easymidi');
```

Comment:
This file seems fine, but there are definitely improvements that could be made:

- Consider splitting out some of the functionality into multiple files so it's easier to manage code (many smaller files instead of one big file) https://webdev-guild.github.io/journeyman/node/modules.html
- Also consider refactoring you code so it is more abstract. In other words, make it so instead of having functions for every page and every selectBtnOnPage, you just have functions for `selectPage(pageNum)` and `selectBtnOnPage(pageNum)`, etc.

Reply:
I've split the code out into modules now.